tool/autoqa/schemaorg/manual-annotations.js

```diff
@@ -189,11 +189,14 @@ const MANUAL_PROPERTY_CANONICAL_OVERRIDE = {
     // restaurants
     'datePublished': {
         passive_verb: ["published | on #", "written | on #"],
-        base: ["date published"]
+        base: ["date published"],
+        adjective_argmax: ['most recent', 'latest', 'last'],
+        adjective_argmin: ['earliest']
```

Comment:
Lol we did not describe these in the paper...
Also, "first", "oldest".

Reply:
added